src/BlazorWebView/src/Maui/Android/WebKitWebViewClient.cs

```diff
@@ -35,13 +37,33 @@ public override bool ShouldOverrideUrlLoading(AWebView? view, IWebResourceReques
 		{
 			// Handle redirects to the app custom scheme by reloading the URL in the view.
 			// Handle navigation to external URLs using the system browser, unless overriden.
-			var requestUri = request?.Url?.ToString();
+			if (request?.Url is { } url)
+			{
+				if (url.Scheme is "http" or "https")
```

Comment:
Is it possible to have a URL with no scheme? Ex. what if the link is `microsoft.com`. Does it get the `http(s)` scheme by default somewhere?

Reply:
If it's something like `<a href="microsoft.com"></a>`, then the URL scheme is implicitly HTTPS. However, in that specific case, the host is also implicitly 0.0.0.0, which makes the link not work anyway.

Reply:
This makes sense.

Related question: are the supplied `Scheme` values guaranteed to be lowercase all the time? Likewise in the `intent` case below.